spec/files/README.md

```diff
@@ -42,3 +42,39 @@ when `patchelf_compatible: true`, and by default fails with `NotImplementedError
 
 ### 2.3) Todo
 Update `why was this added?` after implementing `new_load_method`(also remove this and the last sentence;)
+
+
+## 3. libbuildid.so
+
+### 3.1) Source
+It is generated by the included Makefile.
+
+### 3.2) Why was this added?
+This is a library with the `.gnu.note.build-id` section. It is used to test note sections are correctly relocated along with note segments.
+
+
+## 4. contiguous-note-sections.elf
+
+### 4.1) Source
+It is generated by the included Makefile, using the included contiguous-note-section.ld and contiguous-note-section.s files. The original source can be found at https://github.com/NixOS/patchelf/tree/master/tests.
```

Comment:
use a permalink (with commit hash) because the file might be removed from NixOS/patchelf someday